### DISCUSSION TITLE
Shipping Labels

### DISCUSSION THREAD

**Discussion by f0rmatme** [saleor/saleor#5936] [Ideas]
### What I'm trying to achieve
Shipping Label printing from the order dashboard 

### Describe a proposed solution
Goshippo integration linked to each order, or stamps.com, both have nice APIS 

### Other solutions I've tried and won't work
no solutions 

### Screenshots or mockups
https://i.imgur.com/M79bctE.png

**Comment by f0rmatme** [comment]
Willing to donate up to 100$ BTC if someone is willing to add this to the project i really want to switch from WordPress, this is one feature thats major and not added that would make me switch.

**Comment by KimolO** [comment]
@f0rmatme  I have done an integration of Saleor to https://amphoralogistics.com/  so you can have a 3PL solution that works out of the box (sales ready) like you have in Woo or Shopify. 
Would be nice to have more apps, so we can have a rich environment like Shopify.
